src/components/ThreadEnvelope.vue

```diff
@@ -21,7 +21,12 @@
 
 <template>
 	<div>
-		<div class="icon-mail">
+		<div>
+			<Avatar v-if="envelope.from"
```

Comment:
```suggestion
			<Avatar v-if="envelope.from && envelope.from[0]"
```

Maybe this to be safe?